src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -87,32 +92,27 @@
               :uploadToUrl="LegalServices.uploadToUrl"
             />
           </div>
-        </template>
 
-        <v-divider
-          v-if="isCourtOrder"
-          class="mt-4"
-        />
+          <v-divider class="mt-4" />
+        </template>
 
-        <p
-          v-if="!isAdministerFreeze"
-          class="mt-4"
-        >
-          If this filing is pursuant to a court order, enter the court order number. If this filing is pursuant
-          to a plan of arrangement, enter the court order number and select Plan of Arrangement.
-        </p>
+        <template v-if="!isAdministerFreeze">
```

Comment:
I wrapped both of these in a template with a single v-if, since they go together anyway,